invenio_circulation/api.py

```diff
@@ -152,29 +194,3 @@ def get_loan_for_item(item_pid):
             raise MultipleLoansOnItemError(item_pid=item_pid)
         loan = Loan.get_record_by_pid(hits[0]["pid"])
     return loan
-
-
-def patron_has_active_loan_on_item(patron_pid,
```

Comment:
Method dropped because it was wrong: it was checking for `PENDING` and active states, causing lots of issues when using it.